src/logic/operate.js

```diff
@@ -0,0 +1,27 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let total;
+  const firstNum = Big(numberOne);
+  const secondNum = Big(numberTwo);
+
+  if (operation === '+') {
+    total = firstNum.plus(secondNum);
+  } else if (operation === '-') {
+    total = firstNum.minus(secondNum);
+  } else if (operation === '/') {
+    total = firstNum.div(secondNum);
+  } else if (operation === '*') {
```

Comment:
### Heads-Up:

I think you should use `x` in here cause that is what you're having in `calculate.js`